cloudformation-template.template

```diff
@@ -22,6 +38,52 @@
         "RestApiId": {"Ref": "LanternApiGateway"}
       }
     },
+    "TopAnyMethod": {
+      "Type": "AWS::ApiGateway::Method",
+      "Properties": {
+        "RestApiId": {
+          "Ref": "LanternApiGateway"
+        },
+        "ResourceId": { "Fn::GetAtt": ["LanternApiGateway", "RootResourceId"] },
+        "HttpMethod": "ANY",
+        "RequestParameters": {
+          "method.request.header.x-api-key": true
```

Comment:
FYI, this bit will break existing clients when it goes live - we may need to coordinate and apply the new header to lantern and lantern-dashboard on go live of this